### DISCUSSION TITLE
Welcome to python-trading-robot Discussions!

### DISCUSSION THREAD

**Discussion by areed1192** [areed1192/python-trading-robot#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by regholl** [comment]
I have always had questions about how to operate it but I guess my average knowledge of python prevented me from asking people because sometimes I have gotten answers that made me really feel small,even dumb.Like how to operate it,can i only use one stock at a time,etc.I'm not a total idiot just not experts like you guys so please don't ridicule me.

**Reply by areed1192** [reply]
@regholl Don't ever hesitate to ask a question, that's part of the process. Regarding your question, yes, you can use multiple stocks at a time. If you look at `samples/trading_robot.py` in the repo, it will show you how to specify multiple tickers when you run the program. If you still have questions let me know.

**Comment by DevDiscipline** [comment]
I have a question, I want to start Python but I don't really know where I have to begin.

**Reply by Davidschuck** [reply]
Try a Bing search: Programming for Absolute Beginners pdf. Lots of free books.
First up. Looks ok. https://help.uis.cam.ac.uk/system/files/documents/python3-notes.pdf

**Reply by DevDiscipline** [reply]
Thank you very much! :D

**Comment by cyberfoo0** [comment]
the idea of creating an autonomous AI system that can learn, adapt, and make decisions on its own is a fascinating prospect. the application of AI in stock trading is a rapidly growing field and there's a lot of potential for innovation. what do you think the future of this trading bot is going to be?

**Reply by cyberfoo0** [reply]
are you thinking what I'm thinking :)

**Comment by FaisalWani123** [comment]
Im unable to install the library or any of the packages. Im following the instruction in the README.md file but to no avail. Is anybody else experiencing this?

**Comment by jakedoa** [comment]
I am learning python right now and following the YouTube series that you made about this but ran into some issues. With TD's merger with Schwab, it's not letting me create a new TD Developers account. Any ideas on how to get past this? Also, will that library still work since the TD API is being implemented into Schwab's API?
